src/py/mat3ra/made/tools/build/utils.py

```diff
@@ -78,10 +81,71 @@ def merge_materials(
     distance_tolerance: float = 0.01,
     merge_dangerously=False,
 ) -> Material:
+    """
+    Merge multiple materials into a single material.
+
+    Args:
+        materials (List[Material]): List of materials to merge.
+        material_name (Optional[str]): Name of the merged material.
+        distance_tolerance (float): Distance tolerance to replace close coordinates with the last one, in angstroms.
```

Comment:
The tolerance to replace atoms that are considered too close wrt the coordinates in the last material in the list

Reply:
We replace any existing colliding with those of the last one